functions/java-jre.sh

```diff
@@ -17,7 +17,28 @@ java_webupd8() {
   if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 }
 
-java_zulu_embedded() {
+java_zulu() {
+  FILE="$(mktemp /tmp/.XXXXXXXXXX)"
+
+  if is_arm; then
+    JAVA=zulu8.31.1.122-jdk1.8.0_181-linux_aarch32hf
+  else
+    JAVA=zulu8.31.0.1-jdk8.0.181-linux_x64
+  fi
+  querytext="Downloading Zulu implies to agree to Azul Systems' Terms of Use. Display them now or proceed with downloading."
```

Comment:
Quick check from @kaikreuzer, is copying the terms of use and displaying them here sufficient in your opinion?

Reply:
note  that as part of the install the user also has to confirm he read